h2o-core/src/main/java/water/network/CertAndKeyGenWrapper.java

```diff
@@ -0,0 +1,40 @@
+package water.network;
+
+import sun.security.x509.X500Name;
```

Comment:
No, you were right @mmalohlava IBM Java JDK doesn't contain `sun.security.x509.X500Name`, just tried compiling with it and it failed.

Looking for alternatives.